lib/convert.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const base58 = require('bs58');
+const ed2curve = require('ed2curve');
+
+module.exports = {
+  convertToDhPublicKey(edPublicKeyBase58) {
```

Comment:
TODO: Investigate if it makes sense to move this function to `crypto-ld` (my only hesitation is - not sure how heavy weight `ed2curve` lib is).